### ISSUE TITLE
add instruction on install apisix only and etcd cluster

### ISSUE THREAD

**Issue by Yiyiyimu** [apache/apisix-helm-chart#92]
- install apisix only
https://github.com/apache/apisix-helm-chart/blob/605c0c5520fe2b9b7724df3269b5471f14d88db4/charts/apisix/values.yaml#L215-L220
It seems by default install apisix would not install dashboard and ingress controller together, but when following the guidance, it still installs all three componenents. Tried to override with `helm install apisix apisix/apisix --set dashboard.enabled=false --set ingress-controller.enabled=false` but still failed.

- install etcd cluster
I think it's recommended to install a three-nodes etcd cluster rather than one. Besides, I'm not sure and also failed to find how to passing subchart parameters. 

Hi @tao12345666333 could you help me on this

**Comment by tokers** [comment]
@Yiyiyimu Have you ever tried to clone this repo and use the local helm chart templates?

**Comment by tao12345666333** [comment]
I think this may be due to a problem with the package we published to the Helm repo.

If you clone this repo, you can go through

```
➜ charts git:(master) cd charts
➜ charts git:(master) helm template apisix > /tmp/default.yaml
➜ charts git:(master) helm template apisix --set ingress-controller.enabled=true > /tmp/install-with-apisix-ingress.yaml
````

**Comment by tao12345666333** [comment]
I will submit a PR to update Helm chart version.

**Comment by Yiyiyimu** [comment]
> Have you ever tried to clone this repo and use the local helm chart templates?

Install with local helm chart could success with only apisix and etcd. Waiting for the updates

**Comment by Yiyiyimu** [comment]
> I think this may be due to a problem with the package we published to the Helm repo.

Thanks for the reply and are we fixing this? Another question is that is there any way to install an etcd cluster?

**Comment by tokers** [comment]
> > I think this may be due to a problem with the package we published to the Helm repo.
> 
> Thanks for the reply and are we fixing this? Another question is that is there any way to install an etcd cluster?

You mean just installing ETCD? just use the `bitnami/charts`.

**Comment by Yiyiyimu** [comment]
> You mean just installing ETCD? just use the `bitnami/charts`.

Thanks for the suggestion! Yes I could configure `bitnami/etcd` with [`replicaCount`](https://github.com/bitnami/charts/tree/master/bitnami/etcd#etcd-statefulset-parameters), but failed to find the way to configure it as subchart 😿. Like when I do `helm install apisix .`, which kind of parameter should I use.

**Comment by tokers** [comment]
> > You mean just installing ETCD? just use the `bitnami/charts`.
> 
> Thanks for the suggestion! Yes I could configure `bitnami/etcd` with [`replicaCount`](https://github.com/bitnami/charts/tree/master/bitnami/etcd#etcd-statefulset-parameters), but failed to find the way to configure it as subchart 😿. Like when I do `helm install apisix .`, which kind of parameter should I use.

Please see https://helm.sh/docs/chart_template_guide/subcharts_and_globals/ for the details.

**Comment by Yiyiyimu** [comment]
> Please see [helm.sh/docs/chart_template_guide/subcharts_and_globals](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/) for the details.

I've gone through that page before but it does not mention how to deal with this situation

**Comment by tokers** [comment]
> > Please see [helm.sh/docs/chart_template_guide/subcharts_and_globals](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/) for the details.
> 
> I've gone through that page before but it does not mention how to deal with this situation

Define the required configuration snippet for ETCD subchart in the `values.yaml` in APISIX chart.

**Comment by Yiyiyimu** [comment]
Just realized we are using etcd helm chart in 5.2.1 which has a different schema on `replicaCount` with the latest version ([ref](https://github.com/bitnami/charts/tree/master/bitnami/etcd#to-600)). Bumped to latest in #99 

---

The first problem got fixed after #96
